src/inc/til/small_vector.h

```diff
@@ -871,7 +871,7 @@ namespace til
 
             // An optimization for the most common vector type which is trivially and copyable and noexcept constructible.
             // Compared to the complex form below, we don't need the 2 moves and 1 destroy, because is_trivially_copyable_v implies
-            // that we can just memmove() the items in one fell swoop. We don't need a try/catch either because func() is noexcept.
+            // that we can just memmove() the items at one fell swoop. We don't need a try/catch either because func() is noexcept.

```

Comment:
A coworker complained about people misquoting Shakespeare.

https://github.com/microsoft/terminal/blob/529f938f355f6237a6ecd9a38a0858dfde1fb85b/.github/actions/spelling/line_forbidden.patterns#L66-L70

I'm only changing the text to align with Shakespeare, but this rule actually suggest it should just be:
```suggestion
            // that we can just memmove() the items all at once. We don't need a try/catch either because func() is noexcept.

```

Reply:
all at once is OK for me. I know it's not talking about killing...

(Personally adjusting to `at one fell swoop`)

Reply:
I learned this from a coworker recently.